### PR TITLE
Start updating inspector tests to use Enzyme

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[Start updating inspector tests to use Enzyme](https://trello.com/c/AAKp1ziZ/431-start-updating-inspector-tests-to-use-enzyme)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8588/

File: jujugui/static/gui/src/test/required-props.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ afterEach(function() {
   console.error.calls.all().forEach(call => {
     const error = call.args[0];
     if (error && error.startsWith('Warning: Failed prop type:')) {
-      fail(error);
+      fail(`\x1b[31m${error}\x1b[0m'`);
```
Comment (hatched) [inline jujugui/static/gui/src/test/required-props.js]: Ooo colours

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**